build/integrationtests/Dockerfile

```diff
@@ -7,15 +7,21 @@ LABEL maintainer Rob Bovill <rbovill@lsst.org>
 WORKDIR ${HOME}
 
 ARG integrationtests
+ARG robot_efd
```

Comment:
Consider moving this to line 20. Basically if you leave it here and you change the input docker will rerun the following commands in line 12 and 17 unnecessary recreating layers.